### PR TITLE
Added exclude for google / fb, passowrd etc

### PR DESCRIPTION
**Pull request by artyomtrityak**

### CODE REVIEW

File: server/modules/users/users.controller.js [hunk 1]
```diff
@@ -189,3 +191,24 @@ function createToken(user) {
...
+    return res.status(401).send({
+      message: 'Please make sure your request has an Authorization header'
+    });
+  }
+  var token = req.headers.authorization.split(' ')[1];
+  var payload = jwt.decode(token, config.tokenSecret);
+  if (payload.exp <= moment().unix()) {
+    return res.status(401).send({message: 'Token has expired'});
+  }
+  console.log(payload.sub);
```
Comment (artyomtrityak) [inline server/modules/users/users.controller.js]: will remove

File: server/modules/plays/model.js [hunk 2]
```diff
@@ -37,10 +37,10 @@ var PlaySchema = new Schema({
     type: Schema.Types.ObjectId,
     ref: 'Games'
   },
-  /*players: [{
+  players: [{
```
Comment (Vasso) [inline server/modules/plays/model.js]: why in this case objects array, in other just object?

Comment (artyomtrityak) [inline server/modules/plays/model.js]: there can be multiple players here

File: server/modules/users/users.controller.js [hunk 3]
```diff
@@ -42,7 +42,9 @@ function me(req, res) {
 
 function updateMe(req, res) {
   //TODO : check id from token
-  UserModel.findById(req.user, function (err, user) {
+  //TODO: use findAndUpdate
+  console.log(req.user._id);
```
Comment (Vasso) [inline server/modules/users/users.controller.js]: console.log

**Comment by Vasso** [comment]
cool!

**Pull request merged**